R/derive_vars_computed.R

```diff
@@ -0,0 +1,194 @@
+#' Adds a Variable Computed from the Analysis Value of one or more Parameters
```

Comment:
This function can add multiple variables unless I am mistaken??  Can we update the title to reflect this?  

```suggestion
#' Adds Variable(s) Computed from the Analysis Value of one or more Parameters
```